gammapy/irf/effective_area.py

```diff
@@ -69,6 +69,25 @@ def plot(self, ax=None, energy=None, show_energy=None, **kwargs):
         return ax
 
     @classmethod
+    def from_parametrization(cls, energy, instrument='HESS'):
+        """Get parametrized effective area 
+
+        see :func:`~gammapy.irf.abramowski_effective_area`
```

Comment:
:+1: for having **one** docs page or IPython notebook as an ``gammapy.irf`` tutorial. 

For now I'll leave it in the API docs, and move it to a standalone tutorial later (I will spend some time on writing tutorials in the near future)